### ISSUE TITLE
[BUG] backgound-image of box widget not displayed in i3

### ISSUE THREAD

**Issue by radek-stasta** [elkowar/eww#1190]
### Checklist before submitting an issue

- [X] I have searched through the existing [closed and open issues](https://github.com/elkowar/eww/issues?q=is%3Aissue) for eww and made sure this is not a duplicate
- [X] I have specifically verified that this bug is not a common [user error](https://github.com/elkowar/eww/issues?q=is%3Aissue+label%3Ano-actual-bug+is%3Aclosed)
- [X] I am providing as much relevant information as I am able to in this bug report (Minimal config to reproduce the issue for example, if applicable)

### Description of the bug

I don't exactly know if this is a bug, but I have no idea how to verify it or fix it. I was using XFCE for some time and I have this box for example: 
`
(box
  :width 140
  :height 65
  :style 'background-image: url("https://steamcdn-a.akamaihd.net/steam/apps/379430/header.jpg"); background-size: 100%'
)
`
In XFCE it was working correctly, image was loaded and displayed. Now I switched to i3 (nixos system) and images are gone. I suspect I am missing some package, service, library, something that was present on XFCE and is not present in i3, but I have no idea what. I tried to install all packages listed on eww pages, but still no change

### Reproducing the issue

- install clean i3 system
- create some box widget with background-image from url
- run that widget

### Expected behaviour

Images should be loaded and displayed

**Comment by radek-stasta** [comment]
I managed to get around it with script that downloads files locally and using image widget, but still have no idea why is not working with that bacground-image

**Comment by radek-stasta** [comment]
> reproducible on my nixos system, image does not show up. were you using xfce on nixos as well?

Yes, also on nixos as well. Pretty much same configuration as with i3, don't know about anything special that would allow images to load. Must be something bundled with xfce I think

**Comment by radek-stasta** [comment]
Out of curiosity I tried also on nixos + hyprland, same problem there

**Comment by w-lfchen** [comment]
i am currently using hyprland as well.
i sadly dont really have the time to investigate this currently, sorry.
though i'd offer to test things if someone else picks this up!

**Comment by githubber6800** [comment]
I have discovered on my machine, installing gnome in its entirety fixes it, installing gtk 2,3,4 on their own don't though. It'll be interesting to see if installing gnome fixes it for anyone else and hopefully someone can isolate what specifically does it, so you don't have to have gnome.

I'm using hyprland + nixos, I notice if I add

services.xserver.desktopManager.gnome.enable = true;

it gets fixed by just having it installed, not using it. I'm sure just installing gnome as a package does the same thing.
